xpallets/assets/src/lib.rs

```diff
@@ -271,18 +271,18 @@ impl<T: Config> Module<T> {
 impl<T: Config> Module<T> {
     /// Returns a map of all registered assets by far.
     pub fn total_asset_infos() -> BTreeMap<AssetId, TotalAssetInfo<BalanceOf<T>>> {
-        xpallet_assets_registrar::Module::<T>::asset_infos()
-            .filter_map(|(id, info)| {
+        xpallet_assets_registrar::Pallet::<T>::asset_infos()
+            .filter_map(|(id, _info)| {
                 if id == T::NativeAssetId::get() {
                     // ignore native asset
                     None
                 } else {
                     let data = (
                         id,
                         TotalAssetInfo {
-                            info,
+                            info: xpallet_assets_registrar::AssetInfo::default(),
```

Comment:
If there is no other reason, please do not modify the value arbitrarily, it is very dangerous.

Reply:
this is a problem

Reply:
because it need a `AssetInfo` type